CMakeLists.txt

```diff
@@ -222,6 +221,11 @@ if(QUDA_MAX_MULTI_RHS_TILE GREATER QUDA_MAX_MULTI_RHS)
   message(SEND_ERROR "QUDA_MAX_MULTI_RHS_TILE is greater than QUDA_MAX_MULTI_RHS")
 endif()
 
+set(QUDA_MAX_KERNEL_ARG_SIZE "4096" CACHE STRING "maximum static size of the kernel arguments in bytes passed to a kernel on the target architecture")
```

Comment:
Default for CUDA should be 32,764 bytes not 4096.

Reply:
I set the default value to 4096 here because I'm trying to make the default value the same as before. It was 4096 for both CUDA and HIP when large kernel arg is not enabled. CUDA will always return 32764 if the large kernel arg is enabled whatever value of `QUDA_MAX_KERNEL_ARG_SIZE ` is set.